scripts/temp.deployment.sh

```diff
@@ -21,24 +22,50 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-# Temporary script to deploy argo onto a system for testing
-#     NOTE: this will be removed once we have deployment work done
+set -e
 
-# csm 1.2
+function deployNLS() {
+    BUILDDIR="/tmp/build"
+    mkdir -p "$BUILDDIR"
+    kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d > "${BUILDDIR}/customizations.yaml"
+    kubectl get configmap -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' > "${BUILDDIR}/iuf.yaml"
+    manifestgen -i "${BUILDDIR}/iuf.yaml" -c "${BUILDDIR}/customizations.yaml" -o "${BUILDDIR}/platform.yaml"
+    yq w -i "${BUILDDIR}/platform.yaml" 'spec.charts[0].version' "$2"
+    charts="$(yq r /tmp/build/platform.yaml 'spec.charts[*].name')"
+    for chart in $charts; do
+        if [[ $chart != "cray-iuf" ]] && [[ $chart != "cray-nls" ]]; then
+            yq d -i /tmp/build/platform.yaml "spec.charts.(name==$chart)"
+        fi
+    done
+
+    yq d -i /tmp/build/platform.yaml "spec.sources"
+
+    loftsman ship --charts-path "$1" --manifest-path /tmp/build/platform.yaml
+}
+
+rm -rf /tmp/nls
+git clone https://github.com/Cray-HPE/cray-nls.git /tmp/nls
+if [[ -n $1 ]]; then
+    cd /tmp/nls && git checkout "$1"
+fi
+cd /tmp/nls && helm dep update charts/v1.0/cray-nls/
+cd /tmp/nls && helm package charts/v1.0/cray-nls/
+cd /tmp/nls && helm dep update charts/v1.0/cray-iuf/
+cd /tmp/nls && helm package charts/v1.0/cray-iuf/
+CHART_PATH="/tmp/nls"
+
+echo "Get NLS chart version"
+tarFileName=$(ls -l "$CHART_PATH" | awk '{print $9}' | grep "cray-nls")
```

Comment:
Why use `ls -l` just to get the names of the files? If just trying to get them on one line, it's not necessary. See https://unix.stackexchange.com/questions/295136/pipe-ls-to-grep-lines-vs-items